__tests__/endpoints.test.js

```diff
@@ -82,6 +82,51 @@ describe("Articles", () => {
             });
           });
       });
+      test("should correctly return articles based on query of sort_by and order", () => {
+        return request(app)
+          .get("/api/articles?sort_by=author&order=asc")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles.length > 1).toBe(true);
+            expect(body.articles[0]).toMatchObject({
```

Comment:
It's hard to tell if this is actually working. 
Ideally you would install jest-sorted in order to test the sort and ascending/descending